examples/plot_3d_and_4d_niimg.py

```diff
@@ -0,0 +1,108 @@
+"""
+3D and 4D niimgs: handling and visualizing
+==========================================
+
+Here we discover how to work with 3D and 4D niimgs.
+"""
+
+###############################################################################
+# Downloading tutorial datasets from Internet
+# --------------------------------------------
+#
+# Nilearn comes with functions that download public data from Internet
+#
+# Let's first check where the data is downloaded on our disk:
```

Comment:
> I am afraid that the lambda user can be puzzled by the output (a list
> of paths). Maybe we should add a comment somewhere about that? A link
> to the doc of the function explaining the strategy? Not sure how to
> advertise it.

I can mention that it's a list of path that are tried successively.

On the other hand, on most computers, the list will have only one entry,
so it's not too bad.